### ISSUE TITLE
Injection of variables in array of value producers

### ISSUE THREAD

**Issue by kushan85** [json-template/JsonTemplate#6]
My use case involves an array of value producers and I need to inject unique variables inside each object. Is this possible?

**Comment by haihanyin** [comment]
Do you mean an array of object with unique attribute values? 
Unfortunately this level of randomness is not supported right now. Current version supports randomness on each individual fields and cannot correlates different attributes.
I will work on this issue.

**Comment by kushan85** [comment]
@haihanyin  yes, correct. Also, currently there might be a way to inject data in an array of objects with a custom value producer, but I wasn't able to find good example for that. I will try to play around.
